js/app/controllers/calcontroller.js

```diff
@@ -140,15 +140,18 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
+			constants.publicSharingToken.split(".").forEach(token=>
+			{
+				$scope.calendarsPromise = CalendarService.getPublicCalendar(token).then(function(calendar) {
 				$scope.calendars = [calendar];
+			});
 				is.loading = false;
 				// TODO - scope.apply should not be necessary here
 				$scope.$apply();
-			}).catch((reason) => {
+				}).catch((reason) => {
 				angular.element('#header-right').css('display', 'none');
```

Comment:
Seems to be indent issues here. Does your editor understands [.editorconfig](https://github.com/nextcloud/calendar/blob/master/.editorconfig) files (maybe with a plugin) ?

Reply:
@tcitworld I am using sublime text to edit the code, can you please explain why my pull request can't be merged :)

Reply:
Install this package : https://packagecontrol.io/packages/EditorConfig

There's a few indenting issues I see (when opening a bracket, add a new line indented).

Reply:
I have installed the package and restarted the sublime text but still cannot see any changes made in the indentation of the code.

Reply:
I didn't use Sublime Text for a while, doesn't it shows some errors somehow ?

Don't bother too much for this, I'll fix these kind of typos afterwards.